### DISCUSSION TITLE
can't connect after update to 3.11.0

### DISCUSSION THREAD

**Discussion by robvie** [jamulussoftware/jamulus#3384] [Technical Support & Help] [answered]
I've used Jamulus for years on macbook. After updating to 3.11.0 this week I can't connect nor see any options under "Any genre 1" which is where I'm listed (others can see me & connect). I do see options under all the other genres & can connect into someone else's session. I tried entering my server address as I've had to do in the past, but just get the message "trying to connect." What to do??

**Comment by gilgongo** [comment]
Can you see any Directories at all or is it just "Any genre 1"? Either way, it sounds like a local network issue of some kind (see also [https://explorer.jamulus.io](https://explorer.jamulus.io)). Perhaps reboot your router and/or Macbook? Check to see if some firewall setting hasn't changed?

**Comment by ann0see** [comment]
Otherwise check that the correct input/output device is selected since this could have changed if you didn't transfer the old config file.

**Comment by robvie** [comment]
I can see and connect to every directory under all genres. Only genre 1, where I'm supposed to be is blank.
My clients can see & connect to my directory!

​Rob Viens
Inspirare Association
360.296.1232
[image]

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

**Comment by robvie** [comment]
I can see and connect to every directory under all genres. Only genre 1, where I'm supposed to be is blank.
My clients can see & connect to my directory!

​Rob Viens
Inspirare Association
360.296.1232
[image]

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

**Reply by ann0see** [reply]
Ok. It could well be a fragmentation issue (you can't do much against it, but we're working on a fix - it's not easy to fix unfortunately and probably takes until Jamulus 4). You could only try restarting your router hoping that you get another path to the directory.

**Comment by robvie** [comment, marked as answer]
Fixed!
Discovered I'd downloaded Jamulus server, which was competing with my cloud server!!!

​Rob Viens
Inspirare Association
360.296.1232
[image]

Sent with [Proton Mail](https://proton.me/mail/home) secure email.
